CharlieBackend.Api.UnitTes/MentorServiceTests.cs

```diff
@@ -56,11 +56,13 @@ public async Task CreateMentorAsync()
                 .Callback<Mentor>(x => x.Id = mentorExpectedId);
             _unitOfWorkMock.Setup(x => x.MentorRepository).Returns(mentorRepositoryMock.Object);
 
+            var lessonServiceMock = new Mock<ILessonService>();
+
             var mentorService = new MentorService(
                 _accountServiceMock.Object,
                 _unitOfWorkMock.Object,
                 _mapper,
-                _notificationServiceMock.Object);
+                _notificationServiceMock.Object, lessonServiceMock.Object);
```

Comment:
lessonServiceMock.Object must be on a next line and change variable name to _lessonServiceMock